procedures/foreman/release.md.erb

```diff
@@ -64,22 +64,38 @@ Note: If for some reason there was an issue with the tarballs that required uplo
 - Update [foreman-packaging](https://github.com/theforeman/foreman-packaging) branches
   - [ ] rpm/<%= short_version %> using <%= rel_eng_script('bump_rpm_packaging') %>
   - [ ] deb/<%= short_version %> using <%= rel_eng_script('bump_deb_packaging') %>
+<% if Gem::Version.new(foreman_version) < Gem::Version.new('3.9') -%>
 - [ ] Wait for packages to be built using <%= rel_eng_script('wait_packaging') %> (checks [rpm/<%= short_version %>](https://ci.theforeman.org/job/foreman-packaging-rpm-<%= short_version %>-release/) and [deb/<%= short_version %>](https://ci.theforeman.org/job/foreman-packaging-deb-<%= short_version %>-release/)). This takes a while so you can already start on the manual updates.
+<% else -%>
+- [ ] Wait for packages to be built using <%= rel_eng_script('wait_copr_packaging') %> (checks [rpm/<%= short_version %>](https://ci.theforeman.org/job/foreman-packaging-rpm-copr-<%= short_version %>-release/) and [deb/<%= short_version %>](https://ci.theforeman.org/job/foreman-packaging-deb-<%= short_version %>-release/)). This takes a while so you can already start on the manual updates.
+<% end -%>
```

Comment:
Can this be in one script? :)

Reply:
That is the goal of https://github.com/theforeman/jenkins-jobs/pull/376

Reply:
Perfect

Reply:
https://github.com/theforeman/jenkins-jobs/pull/376 was merged, can you update this PR?